client/src/pages/Dashboard.js

```diff
@@ -14,12 +16,43 @@ const useStyles = makeStyles({
 
 export default function Dashboard() {
   const classes = useStyles();
+  const [openList, setOpenList] = useState(false);
+  const [openAddItem, setOpenAddItem] = useState(false);
+  const [listToDisplay, setListToDisplay] = useState(null);
+
+  const handleClickOpenList = list => {
```

Comment:
I can hopefully use this logic for the notifications/product details dialog as well